content/pandas/concepts/dataframe/terms/resetindex/reset-index.md

```diff
@@ -0,0 +1,168 @@
+---
+Title: '.reset_index()'
+Description: 'This article is about how the .reset_index() is used in pandas dataframes.'
+Subjects: 
+  - 'Data Science'
+  - 'Pandas'
+Tags:
+  - 'Methods'
+  - 'Pandas'
+  - 'Data Science'
+CatalogContent:
+  - 'learn-python'
+  - 'paths/data-science'
+---
+This article is about how the **.reset_index()** is used in pandas dataframes.
+
+
+## Syntax
+```py
+import pandas as pd
+df.read_csv('some_csv.csv', index_col = 'Account Number')
+df.reset_index()
+```
+
+
+### What Is .reset_index() 
+Sometimes there is a need to modify a dataframe object to better meet a certain use case. When these modifications are applied they often leave smaller dataframes with indexes that are messy and not continuous.
+Resetting the index with .reset_index() fixes these issues.
+
+
+### Actions Which Cause Indexing Issues 
+Common examples include but are not limited to:
+
+- Changing the order of columns.
+- Using a column or multiple columns of the database as an index instead of the default index pandas provides and then later needing a numbered index.
+- Filtering out certain rows of the dataframe based on the value of a column of data.
+- Changing the order of rows of data by sorting those rows by the values of a certain column of data such as Date or Employee ID Number.
+- Adding columns and or rows of data to the dataframe that were not present in the original dataframe.
+
+
+### .reset_index() Parameters
+The .reset_index() method provides many parameters which allow for more precision when resetting a dataframe index. The following parameters will be used in the examples later in this article.
+
+- *drop:* The drop parameter can be set to True or False and is set to False by default. When this parameter is set to True it replaces the previous dataframe index with the new index provided by .reset_index() otherwise it sets the new index in front of the old index.
+- *inplace:* The inplace parameter can be set to True or False and is set to False by default. When this parameter is set to True it applies all changes .reset_index() makes to the current instance of the dataframe otherwise it creates a new dataframe instance with the changes applied to that dataframe.
```

Comment:
```suggestion
- *inplace:* The inplace parameter can be set to True or False and is set to False by default. When this parameter is set to True it applies all changes `.reset_index()` makes to the current instance of the DataFrame otherwise it creates a new DataFrame instance with the changes applied to that DataFrame.
```